tools/ci_build/github/linux/docker/build_scripts/finalize.sh

```diff
@@ -20,16 +20,20 @@ for PREFIX in $(find /opt/_internal/ -mindepth 1 -maxdepth 1 -name 'cpython*');
 	if [ -e ${PREFIX}/bin/pip3 ] && [ ! -e ${PREFIX}/bin/pip ]; then
 		ln -s pip3 ${PREFIX}/bin/pip
 	fi
+	PY_VER=$(${PREFIX}/bin/python -c "import sys; print('.'.join(str(v) for v in sys.version_info[:2]))")
```

Comment:
nit: Is `PY_VER` same as `PYVERS` at line 31? (It came from their original code...)

Reply:
I think it should be. But that's their code. :-( 
These two lines are slightly different.